assets/src/BlockEditorValidation.js

```diff
@@ -12,6 +12,7 @@ export const blockEditorValidation = () => {
   subscribe(() => {
     const title = select('core/editor').getEditedPostAttribute('title');
     const featuredImage = select('core/editor').getEditedPostAttribute('featured_media');
+    const postType = select('core/editor').getCurrentPostType();
     const postContent = select('core/editor').getEditedPostContent();
     const blocks = select('core/block-editor').getBlocks();
```

Comment:
Maybe we could make these calls a bit more efficient by avoiding invoking many times functions that fetch the current editor state:

```suggestion
const { getEditedPostAttribute, getCurrentPostType, getEditedPostContent } = select('core/editor');
const { getBlocks } = select('core/block-editor');

const title = getEditedPostAttribute('title');
const featuredImage = getEditedPostAttribute('featured_media');
const postType = getCurrentPostType();
const postContent = getEditedPostContent();
const blocks = getBlocks();
```